presentation/slideshow/4_technologies.md

```diff
@@ -0,0 +1,22 @@
+# Technologies
+
+----------
+
+  * Java 8
+
+  * Docker
+
+  * JavaScript
+
+  	* [Leaflet](https://leafletjs.com)
+  	* [jQuery](https://jquery.com)
+  	* [socket.io](https://socket.io)
+    * [node.js](https://nodejs.org)
+  
+  * Kafka
+  	* [Confluent Schemat Registry](https://docs.confluent.io)
```

Comment:
Probably should be "Schema" instead of "Schemat"

Reply:
JESTEŚMY W POLSCE

Reply:
![joz-rozsierdzon](https://user-images.githubusercontent.com/30010416/40938465-3ca188fa-6842-11e8-9c20-6b99ae17d7a3.jpg)